blog/2024-06-13/index.md

```diff
@@ -0,0 +1,26 @@
+---
+slug: linter-restricting-automatic-merge
+title: "Linter breaking change: restricting automatic merge"
+authors: [barthalion]
+tags: [linter]
+---
+
+Starting next week, June 17th, the linter will no longer allow enabling
+automatic merge for pull requests created by flatpak-external-data-checker,
+unless the app uses an extra-data source or has been added to the exceptions list.
```

Comment:
Should we link to https://docs.flatpak.org/en/latest/module-sources.html#extra-data ?